pyfesom2/fesom2GeoFormat.py

```diff
@@ -0,0 +1,90 @@
+from pyfesom2 import fesom2regular
+from osgeo import osr, gdal
```

Comment:
I would make the osgeo dependency optional. Can you, please, change to something like:
```python
try:
    from osgeo import osr, gdal
except ImportError:
    print('osgeo is not installed, conversion to vector format (fesom2GeoFormat) will not work.')